pkg/manifests/manifests.go

```diff
@@ -1555,10 +1567,12 @@ func (f *Factory) PrometheusK8s(grpcTLS *v1.Secret, trustedCABundleCM *v1.Config
 			p.Spec.Containers[i].Image = f.config.Images.OauthProxy
 
 			f.injectProxyVariables(&p.Spec.Containers[i])
-
 		case "kube-rbac-proxy":
 			p.Spec.Containers[i].Image = f.config.Images.KubeRbacProxy
 			p.Spec.Containers[i].Args = f.setTLSSecurityConfiguration(container.Args, KubeRbacProxyTLSCipherSuitesFlag, KubeRbacProxyMinTLSVersionFlag)
+		case "kube-rbac-proxy-metrics":
```

Comment:
This shall be combined with `case "kube-rbac-proxy"`:

case "kube-rbac-proxy", "kube-rbac-proxy-metrics":
 ...